api-test/DESIGN.md

```diff
@@ -0,0 +1,160 @@
+# Design of the Finesse Test Utility
+
+## Tools available
+
+There are tools that can integrate with Python or a script to accurately
+calculate API statistics. Currently, the needs are to test the tool using JSON
+files containing questions and their page origin in order to establish an
+accuracy score. We also want to calculate request times and generate a
+statistical summary of all this data. That being said, we plan to test the APIs
+under different conditions in the near future. For example, with multiple
+simultaneous users or under special conditions. That's why it's worth
+researching tools, if they are scalable and well adapted with Python.
+
+### Decision
```

Comment:
I can't find the solution you picked in the decision.

Reply:
Corrected by being clear